cdap-ui/app/cdap/components/ScrollableList/index.js

```diff
@@ -47,22 +47,32 @@ export default class ScrollableList extends Component {
     });
   }
   computeHeight() {
-    let targetDimensions = document.getElementById(this.props.target).getBoundingClientRect();
-    let bodyBottom = document.body.getBoundingClientRect().bottom;
-    let targetBottom = targetDimensions.bottom;
-    let heightOfList = bodyBottom - targetBottom;
-    let numberOfElemsInList = Math.floor(heightOfList / 39);
+    let heightOfList = document.body.getBoundingClientRect().height;
+    let numberOfElemsToDisplay = Math.floor(heightOfList / 29);
```

Comment:
Yeah 29 (and before that 39) is/was the height of each item in the submenu. Will add a comment for that